File: templates/python/physics_field_before_after.py
"""physics_field_before_after: 物理场分析前后斜率对比（slope 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='physics field analysis: before-after slope'):
    return make_template_figure('slope', seed=2020, title=title, domain='physics field analysis', topic='before-after slope')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/physics_field_before_after.m
function fig = physics_field_before_after()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('slope', 2020, 'physics field analysis: before-after slope', 'physics field analysis', 'before-after slope');
end
